src/components/SettingsGrid/SettingsGrid.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import React, { PureComponent, Children } from 'react'
+
+type Props = {|
+  +children: Array<React$Node>,
```

Comment:
Array<React$Node> -> React$Node
because React$Node is `Iterable << ? React$Node >` also and can get array of items